template/after.sh.erb

```diff
@@ -1,3 +1,4 @@
+<% if context.cluster != 'kubernetes' -%>
```

Comment:
I think this is now `/kubernetes.*/.match?(context.cluster)` with the test/dev clusters.

Reply:
There are other statements like this in other files too.

Reply:
Fixed to use regex match for `/kubernetes/` where appropriate